### PR TITLE
Consolidating the vcxproj files for the FPGA lsu_control tutorial

### PR DESCRIPTION
**Pull request by aditikum**
## Description

The lsu_control tutorial had separate vcxproj files for different boards. This change consolidates all board types to a single file. This is possible because the argument used for s10 also works for a10 now, whereas it used to generate an error.

## External Dependencies

Works with version oneAPI 2021.3+

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [X] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- [X] Visual Studio

### CODE REVIEW

File: DirectProgramming/DPC++FPGA/Tutorials/Features/lsu_control/lsu_control.sln [hunk 1]
```diff
@@ -17,10 +15,6 @@ Global
 		{D6A634E7-9F2B-46C2-A21C-2402F631A55A}.Debug|x64.Build.0 = Debug|x64
 		{D6A634E7-9F2B-46C2-A21C-2402F631A55A}.Release|x64.ActiveCfg = Release|x64
 		{D6A634E7-9F2B-46C2-A21C-2402F631A55A}.Release|x64.Build.0 = Release|x64
```
Comment (BoleiIntel) [inline DirectProgramming/DPC++FPGA/Tutorials/Features/lsu_control/lsu_control.sln:17]: Should these lines be deleted too?

Comment (aditikum) [reply DirectProgramming/DPC++FPGA/Tutorials/Features/lsu_control/lsu_control.sln:17]: Nope, in this case, those targets have the id: D6A634E7-9F2B-46C2-A21C-2402F631A55A which corresponds to the lsu_control.vcxproj file that is kept.

Comment (BoleiIntel) [reply DirectProgramming/DPC++FPGA/Tutorials/Features/lsu_control/lsu_control.sln:17]: I see. Thanks.

**Review by BoleiIntel — APPROVED** [review]

**Review by mdbtucker — APPROVED** [review]
LGTM, please create a PR to the main repo.

Note that I edited your title and PR description, please use these changes in the PR to the main repo.

**Pull request closed without merging**